ArticleTemplates/articleTemplateContainer.html

```diff
@@ -43,9 +43,8 @@ <h1 class="headline selectable" id="headline">__TITLE__ __REVIEW_RATING__</h1>
         </div>
 
         <div class="tags" id="tags">
-            <ul class="inline-list" id="tag-list">
-                <li class="inline-list__item screen-readable">Tags:</li>
-            </ul>
+            <span class="screen-readable">Tags:</span>
```

Comment:
No I can't think of any reason! This change will however require an update to https://github.com/guardian/mobile-apps-article-templates/blob/master/ArticleTemplates/assets/js/modules/more-tags.js. There's logic in this module to count the tags and only show the first 5 with a 'more' button, which on tap shows the remainder if there are any. This logic takes into account the original "screen-readable" list item jow removed!